### PR TITLE
Use the AWS CLI query and output options for cleaner operation

### PR DESCRIPTION
**Pull request by jdotpz**
Per https://alestic.com/2013/11/aws-cli-query/, I find there is a query option to easily output certain fields of json in the cli.

This PR changes the old dirty sed method to this new clean method.

### CODE REVIEW

**Comment by rhelmer** [comment]
r+ soo much better! 

For bonus points you could break these into multi-line statements with `\` so they fit in 80 chars, would make reading these long lines easier.

File: bin/deploy-socorro.sh [hunk 1]
```diff
@@ -149,11 +149,11 @@ function scale_in_per_elb() {
 function check_health_per_elb() {
     PROGSTEP="Checking ELB status for ${ELBNAME}"
     # We'll want to ensure the number of healthy hosts is equal to total number of hosts
-    ASCAPACITY=`aws autoscaling describe-auto-scaling-groups --auto-scaling-group-names ${AUTOSCALENAME}| grep DesiredCapacity | sed 's/,//g'|awk '{print $2}'`
+    ASCAPACITY=$(aws autoscaling describe-auto-scaling-groups --auto-scaling-group-names ${AUTOSCALENAME} --output text --query 'AutoScalingGroups[*].DesiredCapacity')
         RETURNCODE=$?;error_check
-    HEALTHYHOSTCOUNT=`aws elb describe-instance-health --load-balancer-name ${ELBNAME} | grep InService | wc -l | awk '{print $1}'`
+    HEALTHYHOSTCOUNT=`aws elb describe-instance-health --load-balancer-name elb-stage-socorroweb --output text --query 'InstanceStates[*].State' | grep InService | wc -l | awk '{print $1}'`
```
Comment (rhelmer) [inline bin/deploy-socorro.sh]: +1 on moving to `$()`, looks like you missed this one

**Comment by phrawzty** [comment]
zomg this is so much better, yeah.  We've gone from fish-hands to
yes

**Comment by rhelmer** [comment]
r+

**Pull request merged**